packages/opentelemetry-resources/test/Resource.test.ts

```diff
@@ -84,30 +84,29 @@ describe('Resource', () => {
       'custom.number': 42,
       'custom.boolean': true,
     });
-    assert.equal(resource.attributes['custom.string'], 'strvalue');
-    assert.equal(resource.attributes['custom.number'], 42);
-    assert.equal(resource.attributes['custom.boolean'], true);
+    assert.strictEqual(resource.attributes['custom.string'], 'strvalue');
+    assert.strictEqual(resource.attributes['custom.number'], 42);
+    assert.strictEqual(resource.attributes['custom.boolean'], true);
   });
 
   describe('.empty()', () => {
-    it('should return an empty resource', () => {
+    it('should return an empty resource (except required service name)', () => {
       const resource = Resource.empty();
-      assert.equal(Object.entries(resource.attributes), 0);
+      assert.deepStrictEqual(Object.keys(resource.attributes), []);
     });
 
     it('should return the same empty resource', () => {
       assert.strictEqual(Resource.empty(), Resource.empty());
     });
   });
 
-  describe('.createTelemetrySDKResource()', () => {
-    it('should return a telemetry SDK resource', () => {
-      const resource = Resource.createTelemetrySDKResource();
-      assertTelemetrySDKResource(resource, {
-        language: SDK_INFO.LANGUAGE,
-        name: SDK_INFO.NAME,
-        version: SDK_INFO.VERSION,
-      });
+  describe('.default()', () => {
+    it('should return a default resource', () => {
+      const resource = Resource.default();
+      assert.strictEqual(resource.attributes[ResourceAttributes.TELEMETRY_SDK_NAME], SDK_INFO[ResourceAttributes.TELEMETRY_SDK_NAME]);
+      assert.strictEqual(resource.attributes[ResourceAttributes.TELEMETRY_SDK_LANGUAGE], SDK_INFO[ResourceAttributes.TELEMETRY_SDK_LANGUAGE]);
+      assert.strictEqual(resource.attributes[ResourceAttributes.TELEMETRY_SDK_VERSION], SDK_INFO[ResourceAttributes.TELEMETRY_SDK_VERSION]);
+      assert.strictEqual(resource.attributes[ResourceAttributes.SERVICE_NAME], "@opentelemetry/resources");
```

Comment:
what about service name in browser - using the hostname ?

Reply:
https://github.com/open-telemetry/opentelemetry-js/pull/2227/files#diff-4c2a6d1ab0912f5e733e61c7df71e5d39788a99ab3a1ff9223143f225975af6aR22